src/RuleAction.php

```diff
@@ -646,17 +646,13 @@ public function displayActionSelectPattern($options = [])
                             $display            = true;
                             break;
 
-                        case "dropdown_validation_percent":
-                            $ticket = new Ticket();
-                            echo $ticket->getValueToSelect('validation_percent', 'value', $param['value']);
-                            $display       = true;
-                            break;
-
-                        case "dropdown_validation_status":
-                            TicketValidation::dropdownStatus('value', [
-                                'global' => true,
-                                'value' => $param['value'],
-                            ]);
+                        case "dropdown_percent":
+                            $percentages = range(0, 100, 10); // List of percentages in increments of 10
```

Comment:
It is an unnecessary limitation IMO.

> If someone want to have 1 validation on 3, he has to set 30% and that will do the job.

Having steps of 10 may work well except in cases where there may be more than 10 people that could answer. I can't say for certain there aren't cases where a counsel or working group exists with a larger number of people where a 2/3rds vote (or anything that isn't perfectly Any/50%/100%) is needed for something.

Reply:
ok, I'll change it.

Reply:
I changed it.